src/Api/LiveSyncApiClient.php

```diff
@@ -83,7 +83,7 @@ public function liveSync($shopContents, $shopContentId, $action)
                     'User-Agent' => 'ps-eventbus/' . $this->module->version,
                     'Content-Type' => 'application/json',
                 ],
-                '{"shopContents":' . json_encode($shopContents) . ', "shopContentId": ' . $shopContentId . ', "action": "' . $action . '"}'
+                '{"shopContents":' . json_encode($shopContent) . ', "shopContentId": ' . $shopContentId . ', "action": "' . $action . '"}'
```

Comment:
To avoid calling json_encode, a faster way would be a direct variable interpolation: 

```
php
'{"shopContents": ["' . $shopContent . '"], ..............
```

Reply:
That would mean shopContent is not of type string here?